aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -483,11 +485,21 @@ mod test {
 
     // For compactness, in tests where the Delta contents do not matter.
     fn test_delta() -> DeltaOp {
-        DeltaOp::new(DeltaUpdate::Plus(30), 1000, 30, 0)
+        DeltaOp::new(DeltaValue::Positive(30), 1000, DeltaHistory {
```

Comment:
Let's add a constructor for delta history so that we do not have to add None everywhere